routes/weather.ts

```diff
@@ -259,13 +267,23 @@ export const getWateringData = async function( req: express.Request, res: expres
 			return;
 		}
 
-		wateringData = await weatherProvider.getWateringData( coordinates );
+		try {
+			wateringData = await weatherProvider.getWateringData( coordinates );
```

Comment:
`weatherProvider` base class should define default `getWateringData()` method to raise exception if not overloaded by subclass. That way we wouldn't need L265-269.

Reply:
Resolved in b57b019